### PR TITLE
Enabling GA for regression as well as travis

### PR DESCRIPTION
**Pull request by info-rchitect**

### CODE REVIEW

File: .github/workflows/regression_tests.yml
Comment (priyavadan) [inline .github/workflows/regression_tests.yml]: dont think app generators has web compile

File: .travis.yml
Comment (priyavadan) [inline .travis.yml:1]: I would not comment these out. I probably have to change this in my PR too. We can have both travis and GA enabled simultaneously for a bit

**Review by priyavadan — CHANGES_REQUESTED** [review]

File: .github/workflows/regression_tests.yml
Comment (info-rchitect) [reply .github/workflows/regression_tests.yml]: Yep fixed in latest commit

**Review by priyavadan — APPROVED** [review]

**Comment by priyavadan** [comment]
lets wait on the CI to finish before merging, should pass IMO

**Comment by info-rchitect** [comment]
Merging as per @priyavadan

**Pull request merged**
